index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://kit.fontawesome.com/b957cbbe16.js" crossorigin="anonymous"></script>
+
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="mobile-container">
+    
+    <nav>
+    <div> 
+      <p class="viewmobile">Welcome
+        <i class="fa-solid fa-bars"></i>
+      </p>
+     
+    </div>
+     
+    </nav>
+
+
+  </header>
+  
+    <main class="info">
+      <h2>Hey there. I’m <br> Mandela <br>
+ <span> I’m a software developer </span>
+</h2>
```

Comment:
Kindly change this heading to be `h1` instead of `h2` since this is the main heading of the page.